docs/core/diagnostics/built-in-metrics-runtime.md

```diff
@@ -0,0 +1,242 @@
+
+Available starting in: .NET 9.0.
+
+##### Metric: `dotnet.gc.pause.time`
+
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `dotnet.gc.pause.time` | Counter | `s` | The total amount of time paused in GC since the process has started. |
+
+This metric reports the same values as calling <xref:System.GC.GetTotalPauseDuration?displayProperty=nameWithType>.
+
+Each time the .NET GC does a collection it needs to briefly pause all threads running managed code to determine which objects are still referenced. This metric reports the sum of all these pause times since the process began. This can be used to determine what fraction of time threads spend paused for GC vs. the time they are able to run managed code.
+
+Available starting in: .NET 9.0.
+
+##### Metric: `dotnet.jit.compiled_il.size`
+
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `dotnet.jit.compiled_il.size` | Counter | `By` | Count of bytes of intermediate language that have been compiled since the process has started. |
+
+This metric reports the same values as calling <xref:System.Runtime.JitInfo.GetCompiledILBytes%2A?displayProperty=nameWithType>.
+
+Managed code is initally compiled from a high level language like C#, VB, or F# into [Intermediate language](/standard/managed-code.md#intermediate-language--execution) (IL) when a .NET application is built. Then when the program is run the .NET just-in-time (JIT) compiler converts the IL into machine code.
+
+Since JIT compilation occurs the first time a method runs most JIT compilation tends to occur during application startup. Reducing the amount of IL that is JIT compiled can improve application startup time.
+
+Available starting in: .NET 9.0.
+
+##### Metric: `dotnet.jit.compiled_methods`
+
+| Name     | Instrument Type | Unit (UCUM) | Description    |
+| -------- | --------------- | ----------- | -------------- |
+| `dotnet.jit.compiled_methods` | Counter | `{method}` | The number of times the JIT compiler (re)compiled methods since the process has started. |
+
+This metric reports the same values as calling <xref:System.Runtime.JitInfo.GetCompiledMethodCount%2A?displayProperty=nameWithType>.
+
+Managed code is initally compiled from a high level language like C#, VB, or F# into [Intermediate language](/standard/managed-code.md#intermediate-language--execution) (IL) when a .NET application is built. Then when the program is run the .NET just-in-time (JIT) compiler converts the IL into machine code.
+
+Since JIT compilation occurs the first time a method runs most JIT compilation tends to occur during application startup. Reducing the number of methods that need to be JIT compiled can improve application startup time.
```

Comment:
```suggestion
Since JIT compilation occurs the first time a method runs, most JIT compilation tends to occur during application startup. Reducing the number of methods that need to be JIT compiled can improve application startup time.
```